automation/test.sh

```diff
@@ -148,10 +150,20 @@ if [[ $TARGET =~ os-.* ]] || [[ $TARGET =~ (okd|ocp)-.* ]]; then
     safe_download "$RHEL_LOCK_PATH" "$rhel_image_url" "$rhel_image" || exit 1
 fi
 
-if [[ $TARGET =~ windows.* ]]; then
+if [[ $TARGET =~ windows_sysprep.* ]]; then
+  # Create images directory
+  if [[ ! -d $WINDOWS_SYSPREP_NFS_DIR ]]; then
+    mkdir -p $WINDOWS_SYSPREP_NFS_DIR
+  fi
+
+  # Download Windows image
+  win_image_url="${TEMPLATES_SERVER}/windows2012_syspreped.img"
+  win_image="$WINDOWS_SYSPREP_NFS_DIR/disk.img"
+  safe_download "$WINDOWS_SYSPREP_LOCK_PATH" "$win_image_url" "$win_image" || exit 1
+elif [[ $TARGET =~ windows.* ]]; then
   # Create images directory
-  if [[ ! -d $WINDOWS_NFS_DIR ]]; then
-    mkdir -p $WINDOWS_NFS_DIR
+  if [[ ! -d $WINDOWS_SYSPREP_NFS_DIR ]]; then
```

Comment:
Do we need to change these lines in the elif branch?

Reply:
yes we do. Thank you!